### PR TITLE
updated theme variable name and updated component style.

### PR DESCRIPTION
**Pull request by Shivam-apna**

### CODE REVIEW

File: application/backend/src/main/resources/apps/typerefinery/web_root/light.css [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
-    --primary-background-color: #FFF;
+    --primary-panel-color: #FFF;
     --primary-text-color: #1A2027;
     --subtitle-color: #343a40;
-    --primary-object-background-color: rgb(233 233 233);
+    --primary-bg-color: rgb(233 233 233);
     --ticks-color: #151e26;
     --grid-color: #5D7183;
     --label-color: #072b4a;    
     --border-color: #1a6c93;
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/web_root/light.css:9]: Remove unused variables As mentioned in earlier conversatation

File: application/backend/src/main/resources/apps/typerefinery/web_root/light.css [hunk 2]
```diff
@@ -1,12 +1,11 @@
 :root {
-    --primary-background-color: #FFF;
+    --primary-panel-color: #FFF;
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/web_root/light.css:2]: primary-bg-color for the page backgrounds
card-bg-color for the components background color
border-color

File: application/backend/src/main/resources/apps/typerefinery/web_root/main.css [hunk 3]
```diff
@@ -9,6 +9,15 @@ body {
   width: 100%;
   margin-bottom: 1%;
 }
+
+.lineChart,
+.barCanvas,
+.pieCanvas,
+.xAxisCanvas {
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/web_root/main.css:16]: you can just use one single class

"chartCanvas"

**Review by Harikarthyk — CHANGES_REQUESTED** [review]
Please do a test before giving PR.

I can see few typo corrections using "-border-color" instead of "--border-color".

File: application/backend/src/main/resources/apps/typerefinery/web_root/main.css (hunk 3 above)
Comment (Shivam-apna) [reply application/backend/src/main/resources/apps/typerefinery/web_root/main.css:16]: ok

**Comment by Shivam-apna** [comment]
Please refer #140

**Pull request closed without merging**